tests/EventSystem_1.cc

```diff
@@ -1,7 +1,7 @@
-/*
```

Comment:
Are the C++-based tests at all useful?  Could you dump them in favour of testing everything in python, or are there C++-specific features that you can only test with C++?